pom.xml

```diff
@@ -92,6 +92,7 @@
         <license.copyrightOwners>GReD</license.copyrightOwners>
         <allowedDuplicateClasses>com.esotericsoftware.kryo.*,com.esotericsoftware.minlog.*,com.esotericsoftware.reflectasm.*</allowedDuplicateClasses>
         <soc.version>5.8.3</soc.version>
+        <og.version>5.6.7</og.version>
```

Comment:
Not necessary.

Reply:
done in https://github.com/GReD-Clermont/omero_macro-extensions/pull/10/commits/c76bd5864f85c8febd75677f0d289ae3ec3c4c20